src/components/ui/Navigator.tsx

```diff
@@ -1,19 +1,19 @@
 import Link from "next/link";
 
-interface NavigatorProps {
+interface INavigatorProps {
   projectName: string;
 }
 
-export const Navigator = ({ projectName }: NavigatorProps) => {
-  return (
-    <div className="flex gap-2 text-sm uppercase text-gray-400">
-      <span>
-        <Link href="/projects">Projects</Link>
-      </span>
-      <span>{">"}</span>
-      <span className="text-blue-400">
-        <b>{projectName}</b>
-      </span>
-    </div>
-  );
-};
+export const MyNavigator = ({ projectName }: INavigatorProps): JSX.Element => (
```

Comment:
Why is it called `MyNavigator`?

Reply:
because the linter keeps telling me `Navigator` is already used as global variable

Reply:
You can add it to exceptions for linter or give it a different name (for instance Navigation)

Reply:
That’s why I call it MyNavigator (as a different name). Would you recommend more to add it to exception or change to another name?

Reply:
Let's rename it to Navigation